lib/common.bash

```diff
@@ -170,13 +170,25 @@ clean_env_ctr()
 	local i=""
 	local containers=( $(sudo ctr c list -q) )
 	local count_running="${#containers[@]}"
+	local count_stopped=0
+	local time_out=10
 
 	[ "$count_running" -eq "0" ] && return 0
 
 	for i in "${containers[@]}"; do
-		sudo ctr tasks kill $(sudo ctr task ls | grep $i)
+		sudo ctr tasks kill "$i"
+	done
+
+
+
+	local j=0
+	while [ "$j" -lt "$time_out" ]; do
```

Comment:
I should have looked at it before running the retest, but you could use https://github.com/kata-containers/tests/blob/435563e2dba252d9af217185f58dc52f77e9e75e/.ci/lib.sh#L173 here